core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -776,6 +777,8 @@ public void run() {
                 if (!isCancelled()) {
                     logger.warn("Current thread for topic-partition-id {} is interrupted. Reason: {}", topicIdPartition, ex.getMessage());
                 }
+            } catch (RetriableException ex) {
+                logger.debug("Encountered a retryable error while executing current task for topic-partition {}", topicIdPartition, ex);
```

Comment:
Does `debug` takes throwable as parameter?

Reply:
It does. Sample logs
```
[2023-09-03 12:59:26,937] DEBUG [RemoteLogManager=0 partition=p6WDWrGtTZyCbzgZG_9XBg:Leader-0] Encountered a retryable error while executing current task for topic-partition p6WDWrGtTZyCbzgZG_9XBg:Leader-0 (kafka.log.remote.RemoteLogManager$RLMTask:781)
org.apache.kafka.common.errors.ReplicaNotAvailableException: Remote log metadata cache is not initialized for partition: p6WDWrGtTZyCbzgZG_9XBg:Leader-0
```

I kept the debug message in line with the warning message at L784.